src/components/IGVFileSelector.js

```diff
@@ -41,6 +43,27 @@ export class IGVFileSelector extends Component {
     )(selectedEntities)
   }
 
+  getSelectedFilesList() {
+    const { selectedFiles } = this.state
+    return _.flow(
+      _.keys,
+      _.filter(v => selectedFiles[v])
+    )(selectedFiles)
+  }
+
+  buttonIsDisabled() {
+    const selectedFilesList = this.getSelectedFilesList()
+    return !(selectedFilesList.length > 0 && selectedFilesList.length <= MAX_CONCURRENT_IGV_FILES)
+  }
+
+  getButtonTooltip() {
+    return this.buttonIsDisabled() ? `Select between 1 and ${MAX_CONCURRENT_IGV_FILES} files` : ''
+  }
```

Comment:
nit: These calculations could be inlined into the render method, which would likely improve readability and cut down on extraneous computation a bit.